docs/production/dbos-cloud/otel-integration.md

```diff
@@ -3,7 +3,9 @@ sidebar_position: 75
 title: Export Logs and Traces
 ---
 
-This tutorial shows how to configure your DBOS Cloud application to export OpenTelemetry logs and traces to a third party observability service. We use Datadog as an example. We connect by installing the otel-contrib package in the App VM at deployment time and configuring it with the Datadog API key to export data.
+This tutorial shows how to configure your DBOS Cloud application to export OpenTelemetry logs and traces to a third party observability service. If your service accepts the OTEL format, you can simply add log and traces endpoints to your app config as shown in [step 4](#4-configure-your-app-to-export-logs-and-traces-to-otel-contrib) below. You can pass variables like `OTEL_EXPORTER_OTLP_HEADERS` via your [app secrets](./secrets.md).
```

Comment:
Explicitly say if your service accepts the OTel format, you can skip the first two steps and configure the API key via secrets and pass it into your app.

Reply:
Thanks! In place